src/main/java/mate/academy/springboot/datajpa/service/ProductServiceImpl.java

```diff
@@ -0,0 +1,48 @@
+package mate.academy.springboot.datajpa.service;
+
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.springboot.datajpa.model.Product;
+import mate.academy.springboot.datajpa.repository.ProductRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ProductServiceImpl implements ProductService {
+
+    private final ProductRepository productRepository;
+
+    public ProductServiceImpl(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    @Override
+    public Product save(Product product) {
+        return productRepository.save(product);
+    }
+
+    @Override
+    public Product findById(Long id) {
+        return productRepository.findById(id).orElseThrow(() ->
+                new RuntimeException("Not found product with Id: " + id));
```

Comment:
Advice: after the mentor's review, please check similar places in your code. Why did you change the exception type in `CategoryService`, but didn't change it here?